example/web_socket/echo/client.dart

```diff
@@ -54,7 +54,7 @@ main() async {
     Duration timeout = sockJSTimeout.value.isEmpty
         ? null
         : new Duration(milliseconds: sockJSTimeout.valueAsNumber);
-    var protocols = [];
+    var protocols = <String>[];
```

Comment:
For local variables, yes (see "CONSIDER omitting the types for local variables" section here: https://www.dartlang.org/guides/language/effective-dart/usage).

I'm trying to shift towards using `var` on all local vars that have an immediate assignment since the RHS is enough to appease even the strong-mode analyzer. It keeps it consistent across all variables (LHS is `var name = ` and RHS is typed as specifically as possible) and avoids overly verbose declarations
(`StreamController<List<int>> controller = new StreamController<List<int>>();`).